hack/monitor_prow.py

```diff
@@ -153,5 +158,70 @@ def renderFlavor(pod, dc):
     debug("deployment/{}: pod/{}: got flavor {}".format(dc, pod, lines))
     return lines
 
+
+###Copied from
+###https://docs.google.com/document/d/11Ao_as9Pi2WLIffwn6Xh1XO_857Xra1liAwEVjrGtg8/edit?usp=sharing
+QUERY = '''resource.type="gce_instance"
+(jsonPayload.level="error" OR jsonPayload.level="warning" OR jsonPayload.level="fatal")
+jsonPayload.component!="entrypoint"
+jsonPayload.component!="sidecar"
+jsonPayload.component!="clonerefs"
+jsonPayload.component!="sidecar"
+jsonPayload.component!="initupload"
+jsonPayload.component!="checkconfig"
+jsonPayload.component!="image"
+AND NOT (
+jsonPayload.msg:"Failed to GET ." OR
+jsonPayload.msg:"The following repos define a policy or require context" OR
+jsonPayload.msg:"requested job is unknown to prow: rehearse" OR
+jsonPayload.msg:"requested job is unknown to prow: promote" OR
+jsonPayload.msg:"Not enough reviewers found in OWNERS files for files touched by this PR" OR
+jsonPayload.msg:"failed to get path: failed to resolve sym link: failed to read" OR
+jsonPayload.msg:"nil pointer evaluating *v1.Refs.Repo" OR
+jsonPayload.msg:"unrecognized directory name (expected int64)" OR
+jsonPayload.msg:"failed to get reader for GCS object: storage: object doesn't exist" OR
+jsonPayload.error:"failed to get reader for GCS object: storage: object doesn't exist" OR
+jsonPayload.error:"googleapi: Error 401: Anonymous caller does not have storage.objects.list access to origin-ci-private., required" OR
+jsonPayload.msg:"has required jobs but has protect: false" OR
+jsonPayload.msg:"Couldn't find/suggest approvers for each files." OR
+jsonPayload.msg:"remote error: upload-pack: not our ref" OR
+jsonPayload.msg:"fatal: remote error: upload-pack: not our ref" OR
+jsonPayload.msg:"Error getting ProwJob name for source" OR
+jsonPayload.msg:"the cache is not started, can not read objects" OR
+jsonPayload.msg:"owner mismatch request by" OR
+jsonPayload.msg:"Get : unsupported protocol scheme" OR
+jsonPayload.msg:"No available resource" OR
+jsonPayload.error:"context deadline exceeded" OR
+jsonPayload.error:"owner mismatch request by ci-op"
+)
+
+'''
+
+
+REG_EX = r"AND NOT \(([^}]*)\)"
```

Comment:
Let's just copy the list we care about, as an actual Python list, and not do this

Reply:
Done.